### PR TITLE
Add Zipkin trace configuration into Configuration CRD

### PR DESCRIPTION
**Pull request by tcnghia**
# Description

Add Zipkin trace configuration into Configuration CRD. This is part of https://github.com/dapr/dapr/issues/2337. After docs are updated I will delete the old code in component-contrib.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #2521 #2522 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [x] End-to-end tests passing
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: https://github.com/dapr/docs/issues/967
* [x] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: https://github.com/dapr/docs/issues/978
* [x] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: https://github.com/dapr/docs/issues/967

### CODE REVIEW

File: charts/dapr/crds/configuration.yaml [hunk 1]
```diff
@@ -152,6 +152,12 @@ spec:
                 properties:
                   samplingRate:
                     type: string
+                  zipkin:
+                    type: object
+                    description: ZipkinSpec defines the Zipkin trace configurations
```
Comment (msfussell) [inline charts/dapr/crds/configuration.yaml:157]: Would add a space here   "description: Zipkin spec defines the Zipkin trace configurations"
Or should this be like this? "description: Defines the Zipkin trace configuration settings"

Comment (tcnghia) [reply charts/dapr/crds/configuration.yaml:157]: I like `Defines the Zipkin trace configuration settings`. Updating.

Comment (tcnghia) [reply charts/dapr/crds/configuration.yaml]: Done now

**Review by youngbupark — APPROVED** [review]
lgtm

**Pull request merged**